pipelines/run-collectors/README.md

```diff
@@ -5,14 +5,23 @@ save the required resources to the workspace, execute the collectors, then updat
-| collectorsResource     | The namespaced name of the resource that contains the collectors                                                  | No       | -                                                         |
-| taskGitUrl             | The url to the git repo where the release-service-catalog tasks to be used are stored                             | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
-| taskGitRevision        | The revision in the taskGitUrl repo to be used                                                                    | No       | -                                                         |
+| Name                         | Description                                                                                                       | Optional | Default value                                                |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------|
+| release                      | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution            | No       | -                                                            |
+| previousRelease              | The namespaced name (namespace/name) of the previous successful Release prior to the one passed as params.release | No       | -                                                            |
+| collectorsResourceType       | The type of resource that contains the collectors                                                                 | No       | -                                                            |
+| collectorsResource           | The namespaced name of the resource that contains the collectors                                                  | No       | -                                                            |
+| collectorsRepository         | Git repository where the collectors will be defined                                                               | Yes      | https://github.com/konflux-ci/release-service-collectors.git |
+| collectorsRepositoryRevision | Git repository revision                                                                                           | Yes      | main                                                         |
+| taskGitUrl                   | The url to the git repo where the release-service-catalog tasks to be used are stored                             | Yes      | https://github.com/konflux-ci/release-service-catalog.git    |
+| taskGitRevision              | The revision in the taskGitUrl repo to be used                                                                    | No       | -                                                            |
+
+
+## Changes in 0.4.0
```

Comment:
Why add two versions in one PR? You can just do one new version with two changes

Reply:
My bad. Fixed.